electron/main.ts

```diff
@@ -814,60 +813,50 @@ ipcMain.handle(
       if (updatedRecentGames.length === MAX_RECENT_GAMES) {
         updatedRecentGames.pop()
       }
-      updatedRecentGames.unshift({ appName: game, title })
+      updatedRecentGames.unshift({ appName: game.appName, title })
       configStore.set('games.recent', updatedRecentGames)
     } else {
-      configStore.set('games.recent', [{ appName: game, title }])
+      configStore.set('games.recent', [{ appName: game.appName, title }])
     }
 
     if (minimizeOnLaunch) {
       mainWindow.hide()
     }
 
-    let logResult = ''
-    return Game.get(appName, runner)
-      .launch(launchArguments)
-      .then(async ({ stdout, stderr, command, gameSettings }: LaunchResult) => {
-        logResult = `Launch Command: ${command}
-
-System Info:
+    writeFileSync(
+      game.logFileLocation,
+      `System Info:
 ${await getSystemInfo()}
 
-Game Settings: ${JSON.stringify(gameSettings, null, '\t')}
+Game Settings: ${JSON.stringify(await game.getSettings(), null, '\t')}
+
+Game launched at: ${startPlayingDate}
+
```

Comment:
Don't like the way of writing this multiline strings. There should be a better way like string arrray etc.

E.g.:

```typescript
['first line', 'second line'].join('\n')
```